src/Lighthouse.Tests.Benchmark/Program.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Configuration;
+using Tmds.Utils;
+using Universe.CpuUsage;
+
+namespace Lighthouse.Tests.Benchmark
+{
+    public static class Program
+    {
+        private const int TestLength = 30; // in seconds
+        private const int TestDelay = 5; // in seconds
+        private const int TestRepeat = 20;
+        private const int TestClusterSize = 10;
+        
+        private const string ActorSystemName = "CpuTest";
+        private const string LighthouseAddress = "127.0.0.1";
+        private const int LighthousePort = 54053;
+
+        private static readonly List<CpuUsage> Usages = new List<CpuUsage>();
+        private static readonly List<Process> Processes = new List<Process>();
+
+        public static async Task<int> Main(string[] args)
+        {
+            // ExecFunction hook
+            if (ExecFunction.IsExecFunctionCommand(args))
+                return ExecFunction.Program.Main(args);
+            
+            // Start lighthouse
+            var service = new LighthouseService(LighthouseAddress, LighthousePort, ActorSystemName);

```

Comment:
So we start 1 Lighthouse node in-proc and another remotely via `ExecFunction`?

Reply:
Right now, its one lighthouse in the main benchmark program, and then it will spawn a cluster that uses said lighthose service. Right now it is spawning a cluster with 10 nodes, all using the lighthouse as the seed node.